dbt_subprojects/hourly_spellbook/macros/project/safe/safe_transactions.sql

```diff
@@ -0,0 +1,57 @@
+{% macro safe_transactions(blockchain, project_start_date) %}
+
+-- override param
+{% set project_start_date = '2024-07-31' %}
```

Comment:
Records within the time frame are consistent except for `blast` chain:
https://dune.com/queries/4018595

Reply:
plz do take next steps to dig into blast a bit further on why. is new data more accurate? was there a bug in old code (or present in new code)?

Reply:
Sorry, typo in the param.

Reply:
All matched now:
https://dune.com/queries/4018595